lib/license_finder/reports/text_report.rb

```diff
@@ -2,8 +2,14 @@ module LicenseFinder
   class TextReport < CsvReport
     COMMA_SEP =  ", "
 
-    def initialize(dependencies, options={})
-      super(dependencies, options.merge(columns: %w[name version licenses]))
+    def initialize(dependencies, options)
+      empty_options = {}
+      default_columns = %w[name version licenses]
+      columns = (Array(options[:columns]) & self.class::AVAILABLE_COLUMNS)
+      if columns.empty?
+        columns = default_columns
+      end
+      super(dependencies, empty_options.merge(columns: columns))
```

Comment:
Curious about why the merge here is on `empty_options` and not `options`. It doesn't matter at the moment since `CsvReport` only pays attention to `:columns` but it seems possible to me that in the future `CsvReport` could use other options that would get squashed by replacing `options` with `empty_options`, as happens here. I could be missing something. Is there a reason for using `empty_options`?